### ISSUE TITLE
repairing rope is impossible

### ISSUE THREAD

**Issue by ghasttastic1912** [GabrieleGiuseppini/Floating-Sandbox#85]
when i try to connect peices of rope, the rope vibrates violently and breaks.

**Comment by GabrieleGiuseppini** [comment]
Yes, I know. It's always been an issue. After all, repairing in Floating Sandbox is "live", as in, it happens while materials react to physics. In this case you have an elastic rope that is cut and whose endpoints are being pulled to each other - at some moment the endpoints give in to these elastic forces before they have a chance of getting close to each other.

Will revisit the repair algo one day or the other.

**Comment by ghasttastic1912** [comment]
i suggest making the repair tool stifen the rope somehow although i am not
a gamedev so i dont know how to fix this
i think i am someone who uses the repair tool the most. maybe add an option
to disable rope? /hj
